packages/edit-site/src/components/sidebar-edit-mode/sidebar-card/index.js

```diff
@@ -18,17 +22,19 @@ export default function SidebarCard( {
 } ) {
 	return (
 		<div className={ classnames( 'edit-site-sidebar-card', className ) }>
-			<Icon className="edit-site-sidebar-card__icon" icon={ icon } />
-			<div className="edit-site-sidebar-card__content">
-				<div className="edit-site-sidebar-card__header">
-					<h2 className="edit-site-sidebar-card__title">{ title }</h2>
-					{ actions }
-				</div>
-				<div className="edit-site-sidebar-card__description">
-					{ description }
-				</div>
+			<HStack spacing={ 3 } className="edit-site-sidebar-card__header">
```

Comment:
I'll let designers weight on this. My eye-ruler doesn't like that the icons (title/header vs areas) are not aligned:

| Before | After |
| --- | --- |
| <img width="259" alt="Captura de ecrã 2024-03-13, às 11 01 53" src="https://github.com/WordPress/gutenberg/assets/583546/3b6ff10f-105e-4d38-8b6a-3662edc15dd9"> | <img width="259" alt="Captura de ecrã 2024-03-13, às 11 02 01" src="https://github.com/WordPress/gutenberg/assets/583546/f33622fa-f47f-4b7f-91e3-f0b149903287"> |

Reply:
Happy to make changes in this PR to massage the details so this resembles the designs in https://github.com/WordPress/gutenberg/issues/59689#issuecomment-1991723289, but that might be better handled in a follow-up?

Reply:
Either way works for me